extensions/cql/standard/clause_9_enhanced.adoc

```diff
@@ -220,7 +220,7 @@ CQL follows ISO 8601-1/ISO 8601-2 in defining intervals as closed at both start
 
 Depending on the implementation environment, the underlying datastore may or may not support intervals as data types of properties. If not, intervals are typically represented by two separate properties that are instants, one for the start and one for the end of the interval.
 
-All temporal geometries are in the Gregorian Calendar. This is a deliberate restriction to keep implementations of CQL simple, avoiding requirements to transform time instants to other temporal coordinate reference systems, but still cover a large number of use cases. This is consistent with the use of RFC 3339 as a key standard for expressing date and time on the internet, including in the OGC API standards.
```

Comment:
"temporal coordinate reference system" is correct, "temporal reference system" is outdated and should not be used. See Abstract Spec Topic 2 / ISO 19111.